lib/variable.js

```diff
@@ -91,30 +112,54 @@ module.exports = function getVariableAccessors (container, varName, defValue, ex
     asUrlObject: generateAccessor(require('./accessors/url-object')),
     asUrlString: generateAccessor(require('./accessors/url-string')),
 
+    /**
+     * Instructs env-var to first convert the value of the variable from base64
+     * to utf when reading it using a function such as asString()
```

Comment:
```suggestion
     * when reading it using a function such as asString()
```

(decoding a base64-encoded sequence doesn't necessarily yield bytes that are valid utf-8)